src/components/data/attributetable/attributetable.stories.tsx

```diff
@@ -13,14 +16,42 @@ type Story = StoryObj<typeof meta>;
 export const AttributeTableComponent: Story = {
   args: {
     object: {
-      url: "https://www.example.com",
-      Omschrijving: "Afvalpas vervangen",
-      Zaaktype: "https://www.example.com",
-      Versie: 2,
-      Opmerkingen: null,
-      Actief: false,
-      Toekomstig: false,
-      Concept: true,
+      url: { label: "Url", value: "https://www.example.com" },
```

Comment:
- Can't we use/extend `TypedField` to specify meta data (such as the label) similar to how `DataGrid` handles meta data. Or is there a reason we want to express the label in the object.
- I would appreciate the ability to have a simple key/value datastructure (AttributeData), is this still supported?
- If we need to introduce a new data-structure, can we move it to an appropriate place src/lib/data, an give it such as `LabeledAttributeData` (although I prefer the `TypedField` solution so we can extend the behavior in a more consistent way.

Motivation: I want data components to be "reasonable compatible" so presentation can be easily altered for a certain dataset and API's for individual compoments are relativly predictable.

Reply:
I wanted to specify the label as well because then we can also use react nodes as labels and not just primitives. I am not sure if this can be fit in `TypedField`, but I can add `LabeledAttributeData`.
As it is now it doesn't support the simple key/value structure, but I can add support for it.